lua/default-config.lua

```diff
@@ -1002,6 +1002,7 @@ lvim.lang = {
   },
 }
 
+require("core.autopairs").config()
 require("core.which-key").config()
```

Comment:
Which-key must be configured first because it defines `lvim.builtin.which_key.mappings`, could you add a NOTE so that we don't forget ?

Reply:
Done